src/main/java/gov/va/ascent/framework/rest/provider/RestProviderHttpResponseCodeAspect.java

```diff
@@ -257,19 +264,49 @@ private void writeAudit(Object request, Object response,
         }
 	}
 
-	private void getHttpRequestAuditData(HttpServletRequest httpServletRequest, 
+	private void getHttpRequestAuditData(HttpServletRequest httpServletRequest,
 			RequestResponseAuditData requestResponseAuditData) {
-	    final Map<String, String> headers = new HashMap<>();
+		final Map<String, String> headers = new HashMap<>();
+		Enumeration<String> headerNames = httpServletRequest.getHeaderNames();
 
-        Enumeration<String> headerNames = httpServletRequest.getHeaderNames();
+		while (headerNames.hasMoreElements()) {
+			String headerName = headerNames.nextElement();
+			String value = httpServletRequest.getHeader(headerName);
+			headers.put(headerName, value);
+		}
+		requestResponseAuditData.setHeaders(headers);
+		requestResponseAuditData.setUri(httpServletRequest.getRequestURI());
+		requestResponseAuditData.setMethod(httpServletRequest.getMethod());
+		List<String> attachmentTextList = new ArrayList<>();
+		try {
+			for (Part part : httpServletRequest.getParts()) {
```

Comment:
@rajuthota1 : When testing another end point of GET /document/v1/documentTypes. I saw below errors in the logs. My suggestion is to check if Content-Type is multipart/form-data and then do getParts... 

2018-02-27 12:13:05.734 ERROR [ascent-document-service,f31c4caff18497d4,29b830401cfd4071,true] 90398 --- [nio-8088-exec-5] f.r.p.RestProviderHttpResponseCodeAspect : Error occurred while reading the upload file. {}

javax.servlet.ServletException: org.apache.tomcat.util.http.fileupload.FileUploadBase$InvalidContentTypeException: the request doesn't contain a multipart/form-data or multipart/mixed stream, content type header is null
	at org.apache.catalina.connector.Request.parseParts(Request.java:2909)
	at org.apache.catalina.connector.Request.getParts(Request.java:2777)
	at org.apache.catalina.connector.RequestFacade.getParts(RequestFacade.java:1084)